src/lib/pickles/unfinalized.ml

```diff
@@ -92,3 +92,12 @@ module Constant = struct
     ; sponge_digest_before_evaluations = Digest.Constant.dummy
     }
 end
+
+let typ ~wrap_rounds : (t, Constant.t) Typ.t =
```

Comment:
Does the input typ for statement no longer include this stuff?

Reply:
The goal of this PR was to depend on the input `typ` as an 'output' `typ`. To get this data into the circuit, we need to provide a `typ` for it separate from the input/output one.